pyproject.toml

```diff
@@ -21,7 +21,7 @@ python = "^3.7"
 # Required for Python 3.7 for now. See: https://stackoverflow.com/a/73932581/194311
 importlib-metadata = "4.13.0"
 # Used for local development
-nautobot = "^1.4.0"
+nautobot = { version = "^1.5.0", optional = true }
```

Comment:
this plugin supports nautobot 1.4 as well.

Reply:
.cookiecutter.json should be changed to reflect this as the drift manager relies on it.

Reply:
Fixed to use minimum Nautobot v1.4.0.